hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieHiveCatalog.java

```diff
@@ -245,7 +246,7 @@ public void createDatabase(
 
     Map<String, String> properties = database.getProperties();
 
-    String dbLocationUri = properties.remove(SqlCreateHiveDatabase.DATABASE_LOCATION_URI);
+    String dbLocationUri = properties.remove(DATABASE_LOCATION_URI);
```

Comment:
Can the code compile in lower flink versions before 1.18?

Reply:
Thanks for reviewing the patch. It does not; I missed compiling with the lower Flink version at my local. I am fixing it now and will raise a new revision.